### ISSUE TITLE
[FR] Please add an API

### ISSUE THREAD

**Issue by mimi89999** [Tuhinshubhra/CMSeeK#123]
Hello,
Would it be possible to add an API to make it possible to launch scans from Python code?

**Comment by AnatoleT** [comment]
It could be awesome to add an API

**Comment by AnatoleT** [comment]
@Tuhinshubhra I tried to implement an API with Flask on Heroku. Pretty basic, the api is just getting the url of page, and i execute the command "python3 cmseek.py -o -u" + url
I just don't handled the json path.
Would love to see your work on this API :)

**Comment by Tuhinshubhra** [comment]
@AnatoleT Yeah implementing it isn't much of a challenge. The time needed for it however is something different.
However I'll keep it as a priority.

**Comment by Tuhinshubhra** [comment]
@AnatoleT if you wanna contribute, I'm more than happy to merge that PR. As for me building it tho, not so sure if I really have the time at this point, maybe in the future, sure.
